src/main/java/in/siva/model/ProductDetail.java

```diff
@@ -0,0 +1,66 @@
+package in.siva.model;
+
+public class ProductDetail {
+
+	// Declaring types of details for a product
+	private String productName;
+	private int productPrice;
+	private int productQuantity;
```

Comment:
Suggestions: price instead of ProductPrice. quantity instead of productQuantity.